doc/classes/AcceptDialog.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AcceptDialog" inherits="Window" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Base dialog for user notification.
+		A dialog that can only be accepted or closed.
```

Comment:
`add_button()` disagrees.

Reply:
I took this from the full description, but I suppose it was wrong.